demos/exchange/README.md

```
@@ -0,0 +1,75 @@
+brew install grpcurl
+```
+
+### Injectived
+
+We provide the instruction to build from source and run a local `injectived` node.
+
+Clone `injectived` and `injective-local` in the same directory. `injective-local`
+is used when setting up the node to add some accounts and derivative markets in
+the genesis file. 
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)